content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -77,6 +110,12 @@ Notifications 2.0</td>
 MTM-42808</td>
 </tr>
 
+<tr>
+<td>
+Rest API</td>
+<td>The detail() and delete() method of the @c8y/client MeasurementService has been marked as deprecated as it is not supported with the new timeseries.</td>
```

Comment:
since they are code functions, code markup could be used for "detail()" and "delete()"